packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts

```diff
@@ -12,7 +12,15 @@ import { authConfigToAppSyncAuthType } from '../utils/auth-config-to-app-sync-au
 import { resolverConfigToConflictResolution } from '../utils/resolver-config-to-conflict-resolution-bi-di-mapper';
 import _ from 'lodash';
 import { getAppSyncAuthConfig, checkIfAuthExists, authConfigHasApiKey } from '../utils/amplify-meta-utils';
-import { ResourceAlreadyExistsError, ResourceDoesNotExistError, UnknownResourceTypeError, exitOnNextTick } from 'amplify-cli-core';
+import {
+  ResourceAlreadyExistsError,
+  ResourceDoesNotExistError,
+  UnknownResourceTypeError,
+  exitOnNextTick,
+  stateManager,
+  $TSContext,
+} from 'amplify-cli-core';
+const { doAdminCredentialsExist } = require('amplify-provider-awscloudformation');
```

Comment:
any reason this isn't an import ... from?

Reply:
Yeah, using `import` was breaking things.